compiler/luci/pass/src/FuseMeanWithMeanPass.cpp

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/FuseMeanWithMeanPass.h"
+
+#include <luci/IR/CircleNode.h>
+#include <luci/Profile/CircleNodeOrigin.h>
+
+namespace
+{
+/**
+ *  Fuse two Mean op to one Mean op with merged reduction indices
+ *
+ *  BEFORE
+ *                  |
+ *          [CircleMean, axis<1>]
+ *                  |
+ *         [CircleMean, axis<1>]
+ *                  |
+ *
+ *  AFTER
+ *                  |
+ *          [CircleMean, axis<1,2>]
+ *                  |
+ *
+ */
+luci::CircleConst *create_fused_indices(luci::CircleConst *indices, int32_t index_value,
+                                        int32_t next_index_value)
+{
+  auto name = indices->name();
```

Comment:
Need check if constants  always have names. Does other passes have this kind of asserts?

Reply:
Remove this assert